### PR TITLE
Update ignored files and stats

### PR DESCRIPTION
**Pull request by jonatanklosko**
:cat2: 

Sidenote: are the flake files specific to nixos?

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+/src/**/* linguist-generated=true
```
Comment (the-mikedavis) [inline .gitattributes:1]: oh nice, TIL

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
 /node_modules/
-/build/
+
+# Temporary files
 /tmp/
+
+# Temporary files generated by Tree-sitter
+/build/
+log.html
+tree-sitter-iex.wasm
```
Comment (the-mikedavis) [inline .gitignore:10]: :+1:

**Review by the-mikedavis — COMMENTED** [review]
I actually have part of a similar change brewing locally. Would you mind also:

- removing the `docs` dir (maybe except for `docs/iex.png`?)
- copying over the license from tree-sitter-elixir
    - I think the current one has the appendix which isn't bad but doesn't seem necessary
- removing that flake stuff
    - yep that's all nix stuff. You can actually use it from macos and other linux ditsros though as long as you have `nix` installed. In particular this flake allows me to do `nix develop` and it spawns an ephemeral shell with all the necessary dependencies like nodejs, and when you leave the shell they all disappear :sunglasses: (edit: but I'd say let's remove it, I can work around not having it checked in pretty easily)

File: .gitignore (hunk 2 above)
Comment (jonatanklosko) [inline .gitignore:10]: Or is there any reason to keep it?

**Comment by the-mikedavis** [comment]
So the docs/ dir and `tree-sitter-iex.wasm` are there to drive the github-pages playground page, (e.g. I have another one [here](https://the-mikedavis.github.io/tree-sitter-git-commit/)), which is something cool I originally saw in [tree-sitter-erlang](https://github.com/AbstractMachinesLab/tree-sitter-erlang/)

What do you think about hosting the playground on elixir-lang.github.io/tree-sitter-iex? I could set one up for tree-sitter-elixir too. I like hosting them because they're pretty low maintenance and it helps others get a taste for the grammar without needing to install anything, but it means you need to have that `tree-sitter build-wasm` file checked in and kept up-to-date which can increase the repo size which isn't ideal

File: package.json [hunk 3]
```diff
@@ -3,28 +3,20 @@
   "version": "0.0.1",
   "description": "A tree-sitter grammar for Elixir's \"IEx\" REPL syntax",
```
Comment (the-mikedavis) [inline package.json]:
```suggestion
  "description": "Interactive Elixir (IEx) grammar for tree-sitter",
```

I like the new repo description text better than the old. What do you think?

**Comment by jonatanklosko** [comment]
@the-mikedavis Thanks! Hosting the playground sounds good, but ideally we would keep it on a separate branch and have CI update there, while keeping those files off the main branch. There shouldn't be any obstacles to that, right?

**Comment by the-mikedavis** [comment]
ah even better, :shipit:!

**Review by the-mikedavis — APPROVED** [review]
:confetti_ball:

File: .gitattributes (hunk 1 above)
Comment (the-mikedavis) [reply .gitattributes:1]: @jonatanklosko what do you think about marking the `bindings/` dir as generated as well? looks like linguist currently thinks tree-sitter-iex is a rust project :smile:

Comment (jonatanklosko) [reply .gitattributes:1]: Haha, nice one, will do!

**Pull request merged**